tests/test_mmd_to_iso.py

```diff
@@ -1,57 +1,25 @@
 import tempfile
+import os
 import unittest
-import confuse
 from py_mmd_tools.mmd_to_csw_iso import mmd_to_iso
 
 
 class test_pymmdtools(unittest.TestCase):
     def setUp(self):
-        # Note:
-        # The "confuse" library is used to pass the path string for static files
-        # used to perform the test.
+        # run this test from the tests directory by running:
         #
-        # On linux systems, by default, it reads a configuration file from:
-        # $HOME/.config/appname/config.yaml (in this case appname=mdtool)
+        # python3 test_mmd_to_iso.py
         #
-        # The confuse configurationfile uses yaml syntax, example:
+        # or from the root directory by running:
         #
-        # """
-        # paths:
-        #     reference_mmd: 'path/to/reference_mmd.xml'
-        #     reference_iso: 'path/to/reference_iso.xml'
-        #     mmd2isocsw: 'path/to/mmd-to-iso.xsl'
-        # """
-        #
-        # -- see the confuse documentation for more details. --
-        #
-        #
-        # If you do not want (or can not) use the 'confuse' library,
-        # replace the values of:
-        #
-        # reference_mmd
-        # reference_iso
-        # mmd2iso_xslt
-        #
-        # with the path strings to the files provided by your testing environment, and comment/remove the following lines:
-        #
-        # """
-        # import confuse
-        # config = confuse.Configuration('mmdtool', __name__)
-        # """
-        #
-        # example file used in this test can be found at:
-        # https://gist.github.com/934db4d3cf4e7a52985a3e231e0d36cf
-        #
-        # run this test from the module directory by running:
-        #
-        # python3 test_mmdtoiso.py
+        # python3 -m unittest 
         #
         # unset the output limit when printing the xml diff
         #
-        config = confuse.Configuration('mmdtool', __name__)
-        self.reference_mmd = config['paths']['reference_mmd'].get()
-        self.reference_iso = config['paths']['reference_iso'].get()
-        self.mmd2iso_xslt = config['paths']['mmd2isocsw'].get()
+        current_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
we may want to use pathlib for path handling, something like:

```python
import pathlib
current_dir = pathlib.Path.cwd()
self.reference_mmd = current_dir / 'data' / 'reference_mmd.xml'
self.reference_iso = current_dir / 'data' /  'reference_iso.xml'
self.mmd2iso_xslt = current_dir / 'data' / 'mmd-to-iso.xsl'

Reply:
I agree that pathlib is better than os.
But what's the best way afterwards? The `ET.ElementTree` function does not like PosixPath and wants a string instead. I can do sthg like that? But I don't find it very nice.

```python
        mmd_to_iso(
            mmd_file=str(self.reference_mmd),
            outputfile=tested,
            mmd2isocsw=str(self.mmd2iso_xslt),
        )

Reply:
I see the problem, right. I guess it could be better to  cast the path to be a string at the very begin :
```python
import pathlib
current_dir = pathlib.Path.cwd()
self.reference_mmd = str(current_dir / 'data' / 'reference_mmd.xml')
self.reference_iso = str(current_dir / 'data' /  'reference_iso.xml')
self.mmd2iso_xslt = str(current_dir / 'data' / 'mmd-to-iso.xsl')
```

Reply:
Ok. I'll do that.